spec/helpers/application_helper/toolbar_chooser_spec.rb

```diff
@@ -172,5 +172,12 @@
         expect(ApplicationHelper::ToolbarChooser.new(nil, nil, :record => @record, :explorer => true, :layout => 'automation_manager', :sb => @sb).send(:x_view_toolbar_filename)).to eq("x_gtl_view_tb")
       end
     end
+
+    context 'when in bottlenecks explorer' do
+      it "displays toolbar" do
+        @sb = {:active_tab => 'summary'}
+        expect(ApplicationHelper::ToolbarChooser.new(nil, nil, :record => @record, :explorer => true, :layout => 'miq_capacity_bottlenecks', :sb => @sb).send(:x_view_toolbar_filename)).to eq("miq_capacity_view_tb")
+      end
```

Comment:
There's a simplew way to chect that a particula toolbar is instantiated when a given action is requested.

Example from `spec/controllers/physical_switch_controller_spec.rb`:
```
        expect(ApplicationHelper::Toolbar::PhysicalSwitchesCenter).to receive(:definition).and_call_original
```

Not sure if it's applicable to your situation but generally it does not need much mocking.